Gordon360/Controllers/ProfilesController.cs

```diff
@@ -476,14 +480,28 @@ public async Task<ActionResult<FacultyStaffProfileViewModel>> UpdateOfficeHours(
             return Ok(result);
         }
 
+        /// <summary>
+        /// Update mail location
+        /// </summary>
+        /// <param name="value">mail location</param>
+        /// <returns></returns>
+        [HttpPut]
```

Comment:
Maybe the route should be "mailstop" or "mailstop_update"...

Reply:
I don't think this is necessary: the `put` implies the updating of the resource `mail_stop`. changing this to `mailstop_update` would be implying that we are updating a mailstop_update